tests/test_lightning_question_answering_pipeline.py

```diff
@@ -0,0 +1,111 @@
+from pathlib import Path
+from typing import Any, Dict
+
+import numpy as np
+import pytest
+import pytorch_lightning as pl
+import torch
+from _pytest.tmpdir import TempdirFactory
+
+from embeddings.config.lightning_config import LightningAdvancedConfig
+from embeddings.evaluator.evaluation_results import QuestionAnsweringEvaluationResults
+from embeddings.pipeline.hf_preprocessing_pipeline import HuggingFacePreprocessingPipeline
+from embeddings.pipeline.lightning_question_answering import LightningQuestionAnsweringPipeline
+from embeddings.task.lightning_task.question_answering import QuestionAnsweringTask
+
+
+@pytest.fixture(scope="module")
+def tmp_path_module(tmpdir_factory: TempdirFactory) -> Path:
+    path = tmpdir_factory.mktemp(__name__)
+    return Path(path)
+
+
+@pytest.fixture(scope="module")
+def dataset_kwargs() -> Dict[str, Any]:
+    return {
+        "dataset_name_or_path": "./data_sample"
+    }
+
+
+@pytest.fixture(scope="module")
+def config() -> LightningAdvancedConfig:
+    return LightningAdvancedConfig(
```

Comment:
I would change it to newly created `QuestionAnsweringLightningBasicConfig` or `QALightningBasicConfig` instead of passing as `Advanced` one